src/apolo_app_types/protocols/hugging_face.py

```diff
@@ -1,21 +1,31 @@
 from pydantic import Field
 
 from apolo_app_types.protocols.common import AppInputs, AppOutputs, HuggingFaceCache
+from apolo_app_types.protocols.common.hugging_face import HuggingFaceToken
 from apolo_app_types.protocols.common.schema_extra import (
     SchemaExtraMetadata,
     SchemaMetaType,
 )
 
 
-class HuggingFaceCacheInputs(AppInputs):
+class HuggingFaceAppInputs(AppInputs):
     cache_config: HuggingFaceCache = Field(
         json_schema_extra=SchemaExtraMetadata(
             title="Hugging Face Cache",
             description="Configuration for the Hugging Face cache.",
             meta_type=SchemaMetaType.INLINE,
         ).as_json_schema_extra()
     )
+    tokens: list[HuggingFaceToken] = Field(
```

Comment:
why do we need list of tokens? I mean, how can we then consume them and differentiate which model uses which token